app/policies/pageflow/account_policy.rb

```diff
@@ -123,13 +124,13 @@ def destroy_membership_on?
     end
 
     def admin?
-      @user.admin?
+      user.admin?
     end
 
     def see_badge_belonging_to?
-      (@account.entries & @user.entries).any? ||
-        @user.memberships.on_accounts.as_previewer_or_above.where(entity: @account).any? ||
```

Comment:
(We would need `on_as_many_accounts_as: 2` for now, but doing it for `1` and `2` only seems excessively lazy)